### DISCUSSION TITLE
tine podcast: 18 – Das Adressbuch

### DISCUSSION THREAD

**Discussion by pschuele** [tine-groupware/tine#23] [Announcements]
Basics, Teil 2: Nach dem Kalender kommt diesmal das Adressbuch unter die Lupe. Keine Sorge, staubig wird es nicht, schließlich hat das Adressbuch seit „Ellie“ einen moderneren Anstrich bekommen. Philipp Schüle erklärt die neuen Optionen bei den Feldern, berichtet von einigen weniger bekannten Features und natürlich darf auch der Ausblick auf kommende Entwicklungen nicht fehlen, denn schließlich ist das Adressbuch in tine universeller, als man im ersten Augenblick denken mag.

Gebt uns gerne Feedback zu dieser Episode. Habt Ihr vielleicht Themen, über die wir unbedingt mal sprechen sollten oder andere Anregungen? Wenn es Euch gefallen hat, hinterlasst uns doch eine positive Bewertung im Podcast-Portal Eurer Wahl.

Neue Episoden erscheinen immer am vierten Mittwoch des Monats – am besten richtet Ihr Euch schon mal einen Serientermin in tine ein.

**Comment by lab-at-nohl** [comment]
Hier, ich habe ein Thema :-) Ich habe immer das Problem, dass ich Dubletten in verschiedenen Ordnern habe. Kann man da was machen mit der neuen Flexibilität - also ein Kontakt aber zwei Orte? 

Der Hintergrund ist banal und auch etwas doof. Ich synchronisiere nur einen bestimmten Ordnern mit dem Handy (ActiveSync). Früher hatte ich dafür einen Sync-Filter, der konnte den Ordner (auch für neu angelegte Kontakte) und ein Tag "Handy". Leider funktioniert das mit den Tags nicht (mehr), ich denke weil der Google Client einerseits Ordner unterstützt (FolderSync), andererseits Tine in diesem Falle aber nicht mehr den Sync-Filter, sondern den Ordner anbietet. Ich müsste also einen Kontakt in Tine irgendwie mit dem Ordner Handy "verknüpfen". 

Ansonsten, ich bin treuer Fan eures Podcasts. Finde das sehr angenehm, muss man nichts dazu lesen.

**Reply by pschuele** [reply]
moin Johannes, danke für die rückmeldung - freut mich :smile: 

das mit den dubletten wirklich etwas doof ... eigentlich hätte ich auch gehofft, dass man dem device über den sync-filter genau mitteilen kann, welche kontakte man im sync haben möchte. das wäre meiner meinung nach der beste weg, das problem zu lösen. bin leider nicht so tief drin im Android ActiveSync, dass ich direkt sagen könnte, woran es liegt und ob man das fixen kann.

vielleicht machst du mal ein issue auf? ich spreche das dann mit den kolleg*innen an...
